features/FooterUpdate.feature

```diff
@@ -0,0 +1,66 @@
+@owner:derek
+@author:lilla
+Feature: Docs Footer Update
+
+    New responsive design and links for the footer on the docs pages.
+
+    Background: A user arrives on the SaaSquatch help center page
+        Given the user stays on the 'https://docs.saasquatch.com/' pages
+        Then the footer categories are visible
```

Comment:
This isn't really needed in the background since your first specification covers this